src/x_misc.c

```diff
@@ -559,7 +559,15 @@ static void oscformat_list(t_oscformat *x, t_symbol *s, int argc, t_atom *argv)
             typecode = 's';
         else typecode = 'f';
         if (typecode == 's')
-            msgindex += ROUNDUPTO4(strlen(argv[j].a_w.w_symbol->s_name) + 1);
+        {
+            if (argv[j].a_type == A_SYMBOL)
+                msgindex += ROUNDUPTO4(strlen(argv[j].a_w.w_symbol->s_name) + 1);
+            else
+            {
+                pd_error(x, "oscformat: expected symbol but got float");
```

Comment:
iirc this error message is consistent with similar errors (even though the "float" might also be a "gpointer").
probably a better wording would be `"oscformat: expected symbol as argument #%d" % (j,))`

Reply:
I thought of that, but the types that oscformat takes are only floats and symbols anyway. The rest get coerced into a float.

Reply:
I would use "oscformat: expected symbol for argument %d" but should the index print be 0 or 1-indexed? ... :)

Reply:
Done. 1-indexed makes sense in this context.